animation_nodes/nodes/falloff/mesh_falloff.pyx

```diff
@@ -0,0 +1,73 @@
+import bpy
+import cython
+from mathutils import Vector
+from ... math cimport Vector3
+from ... utils.bvh import isInsideVolume
+from ... base_types import AnimationNode
+from ... data_structures cimport BaseFalloff
+from . constant_falloff import ConstantFalloff
+
+class MeshFalloffNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_MeshFalloffNode"
+    bl_label = "Mesh Falloff"
+
+    def create(self):
+        self.newInput("BVHTree", "BVHTree", "bvhTree")
+        self.newInput("Float", "Size", "size")
+        self.newInput("Float", "Falloff Width", "falloffWidth", value = 1)
+        self.newInput("Boolean", "Use Surface", "useSurface", value = True)
```

Comment:
`Use Surface` can be removed, as in, it should always be True. A False `Use Surface` and a True `Use Volume` is equivalent to a True `Use Surface` and a True `Use Volume` when Scale and Falloff Width are both zero. You may check for this condition to avoid extra computations implicitly. Moreover, `Use Volume` should be renamed to something more clear, maybe `Fill Inside`.

Reply:
Okay.